templates/validator/charts.html

```diff
@@ -236,8 +236,10 @@
                                 const startEpoch = timeToEpoch(tooltip.chart.hoverPoint.x)
                                 const timeForOneDay = 24 * 60* 60 * 1000
                                 const endEpoch = timeToEpoch(tooltip.chart.hoverPoint.x + timeForOneDay) - 1
-                                const endTs = epochToTime(endEpoch)
-                                text += `${new Date(tooltip.chart.hoverPoints[0].x).toLocaleDateString()} - ${new Date(endTs).toLocaleDateString()} <br> Epochs ${startEpoch} - ${endEpoch}<br/>`;
+                                const startDate = luxon.DateTime.fromMillis(tooltip.chart.hoverPoints[0].x)
+                                const endDate = luxon.DateTime.fromMillis(epochToTime(endEpoch))
+                                const endTime = luxon.DateTime.fromMillis(epochToTime(endEpoch+1))
+                                text += `${startDate.toFormat("MMM-dd-yyyy HH:mm:ss")} - ${endDate.toFormat("MMM-dd-yyyy")} ${endTime.toFormat("HH:mm:ss")} <br> Epochs ${startEpoch} - ${endEpoch}<br/>`;
```

Comment:
Why do we have `endDate` and `endTime`?
Isn't 
```suggestion
                                const startDate = luxon.DateTime.fromMillis(tooltip.chart.hoverPoints[0].x)
                                const endDate = luxon.DateTime.fromMillis(epochToTime(endEpoch+1))
                                text += `${startDate.toFormat("MMM-dd-yyyy HH:mm:ss")} - ${endDate.toFormat("MMM-dd-yyyy HH:mm:ss")} <br> Epochs ${startEpoch} - ${endEpoch}<br/>`;
```
always correct?

Reply:
fixed